device/bgp/bgpv6.yaml

```diff
@@ -30,7 +30,7 @@ components:
           type: string
           x-constraint:
           - '/components/schemas/Device.Ipv6/properties/name'
-          - '/components/schemas/Device.LoopbackIpv6/properties/name'
+          - '/components/schemas/Device.Ipv4Loopback/properties/name'
```

Comment:
Device.Ipv4Loopback --> Device.Ipv6Loopback?

Reply:
Corrected. Thank you